website/content/en/preview/getting-started/migrating-from-cas/_index.md

```diff
@@ -92,10 +92,10 @@ One for your Karpenter node role and one for your existing node group.
 First set the Karpenter release you want to deploy.
 
 ```bash
-export KARPENTER_VERSION={{< param "latest_release_version" >}}
+export KARPENTER_VERSION="{{< param "latest_release_version" >}}"
```

Comment:
Does it make sense to just change this version to not have the `v` and then append the `v` onto the version when we need it, like when we are pulling raw files from Github?

Reply:
That was my original plan, so I'm happy to pivot to this.